cas/cas_test.go

```diff
@@ -449,5 +449,99 @@ func TestTreeStore(t *testing.T) {
 	if err == nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
+}
+
+func TestRemoveACI(t *testing.T) {
+	dir, err := ioutil.TempDir("", tstprefix)
+	if err != nil {
+		t.Fatalf("error creating tempdir: %v", err)
+	}
+	defer os.RemoveAll(dir)
+	ds, err := NewStore(dir)
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
+
+	imj := `{
+                    "acKind": "ImageManifest",
+                    "acVersion": "0.4.0",
+                    "name": "example.com/test01"
+                }`
+
+	aciFile, err := aci.NewACI(dir, imj, nil)
+	if err != nil {
+		t.Fatalf("error creating test tar: %v", err)
+	}
+	// Rewind the ACI
+	if _, err := aciFile.Seek(0, 0); err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
+	key, err := ds.WriteACI(aciFile, false)
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
```

Comment:
The code above is a copypasta from TestGetImageManifest. Consider putting it in common function.

Or maybe postpone it - looks like the ACI preparation in this file really could use some refactoring.